[enterprise 4-18] Release notes for telco features (#88257)
openshift/openshift-docs: release_notes/ocp-4-18-release-notes.adoc

For more information, see xref:../backup_and_restore/graceful-cluster-shutdown.adoc#graceful-shutdown-cluster[Shutting down the cluster gracefully].

[id="ocp-4-18-deprecation-of-siteconfig-v1_{context}"]
==== Deprecation of SiteConfig v1

> SiteConfig v1 is deprecated starting with {product-title} version 4.18. Equivalent and improved functionality is now available through the SiteConfig Operator using the `ClusterInstance` custom resource. For more information, see link:https://access.redhat.com/articles/7105238[Procedure to transition from SiteConfig CRs to the ClusterInstance API].

Reviewer: ```suggestion
SiteConfig v1 is deprecated starting with {product-title} 4.18. Equivalent and improved functionality is now available through the SiteConfig Operator using the `ClusterInstance` custom resource. For more information, see the Red Hat Knowledge Base solution link:https://access.redhat.com/articles/7105238[Procedure to transition from SiteConfig CRs to the ClusterInstance API].
```

Reviewer: https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values-xml:~:text=For%20a%20non%2Dcloud%20environment%2C%20you%20can%20resize%20the%20disk%20and%20file%20system.%20For%20more%20information%2C%20see%20the%20Red%C2%A0Hat%20Knowledgebase%20solution%20Does%20RHEL%207%20support%20online%20resize%20of%20disk%20partitions%3F.

Author: Updated as per the suggestion.

> SiteConfig v1 is deprecated starting with {product-title} 4.18. Equivalent and improved functionality is now available through the SiteConfig Operator using the `ClusterInstance` custom resource. For more information, see the Red Hat Knowledge Base solution link:https://access.redhat.com/articles/7105238[Procedure to transition from SiteConfig CRs to the ClusterInstance API].

Reviewer 2: https://redhat-documentation.github.io/supplementary-style-guide/#non-breaking-spaces

> Use a non-breaking space ({nbsp}) between the words "Red" and "Hat". The non-breaking space prevents an automatic line break from separating the two words onto two lines.

[id="ocp-release-notes-networking_{context}"]
=== Networking

[id="ocp-4-18-holdover-in-grandmaster-clock_{context}"]
==== Holdover in a grandmaster clock with GNSS as the source

> With this release, you can configure the holdover behavior in a grandmaster (T-GM) clock with GNSS as the source. Holdover allows the T-GM clock to maintain synchronization performance when the GNSS source is unavailable. During this period, the T-GM clock relies on its internal oscillator and holdover parameters to reduce timing disruptions.

Reviewer: You might want to spell our GNSS the first time you use it.

Author: Updated as per the suggestion.

For more information about the `cluster-compare` plugin, see xref:../scalability_and_performance/cluster-compare/understanding-the-cluster-compare-plugin.adoc#cluster-compare-overview_understanding-cluster-compare[Overview of the cluster-compare plugin].

[id="ocp-4-18-support-for-worker-nodes-on-amd-cpus_{context}"]

> ==== Support for Worker Nodes with AMD EPYC Zen 4 CPUs

Author: Approved by @MarSik in slack!

For more information, see xref:../scalability_and_performance/using-node-tuning-operator.adoc#defer-application-of-tuning-changes_node-tuning-operator[Deferring application of tuning changes].

[id="ocp-4-18-support-for-worker-nodes-on-amd-cpus_{context}"]
==== Support for Worker Nodes with AMD EPYC Zen 4 CPUs

> With this release,  you can deploy worker nodes on machines equipped with AMD EPYC Zen 4 CPUs (Genoa, Bergamo, or newer). These CPUs are fully supported. However, the power consumption evaluations are ongoing. It is recommended to evaluate features, such as per pod power management, to determine any potential impact.

Reviewer 3: ```suggestion
With this release,  you can use PerformanceProfiles to configure worker nodes on machines equipped with AMD EPYC Zen 4 CPUs (Genoa and Bergamo). These CPUs are fully supported. However, the power consumption evaluations are ongoing. It is recommended to evaluate features, such as per pod power management, to determine any potential impact.
```

The or newer is too much at this time. Turin is probably going to work, but no testing has been done yet. The official cpu list supported by OCP should be the main source of truth. Since this is Telco specific release note, lets make that explicit by mentioning the PerfProfile.

Author: @MarSik 
Updated as follows:

With this release, you can use the `PerformanceProfile` custom resource (CR) to configure worker nodes on machines equipped with AMD EPYC Zen 4 CPUs (Genoa and Bergamo). These CPUs are fully supported. However, the power consumption evaluations are ongoing. It is recommended to evaluate features, such as per pod power management, to determine any potential impact.

* AArch64
* Intel 64

[id="ocp-4-18-support-for-worker-nodes-on-amd-cpus_{context}"]

> ==== Support for Worker Nodes with AMD EPYC Zen 4 CPUs

Reviewer: ```suggestion
==== Support for worker nodes with AMD EPYC Zen 4 CPUs
```

Author: Updated as per the suggestion.
